.github/workflows/nightly.yml

```diff
@@ -0,0 +1,28 @@
+name: nightly
+on:
+  schedule:
+    # runs every day at 6am UTC
+    - cron: "0 6 * * *"
```

Comment:
lol ~nightly~ daily

Reply:
should we leave it here? or I remove it?

Reply:
I think we can keep it in this PR. I don't see a reason to intentionally avoid backporting it.